features/upgrade/storage/upgrade.feature

```diff
@@ -0,0 +1,99 @@
+Feature: basic verification for upgrade testing
+  # @author chaoyang@redhat.com
+  #  @upgrade-prepare
+  #@users=upuser1,upuser2
+  @admin
+  Scenario: Snapshot operator should be in available status after upgrade and can created pod with snapshot
+    Given I switch to cluster admin pseudo user		
+    When I run the :create admin command with:
+      | f | <%= BushSlicer::HOME %>/testdata/storage/csi/aws-ebs-with-snapshots.yaml |
+    Then the step should succeed
+    When I use the "kube-system" project
+    And I wait up to 180 seconds for the steps to pass:
+    """
+    When I run the :describe client command with:
+      | resource | deployment         |
```

Comment:
@chao007 any reason behind using `oc describe` instead of `oc get`?

Reply:
In order to get output like this  `1 desired.*1 updated.*1 total.*1 available.*0 unavailable`